frontend/pages/hosts/details/cards/HostSummary/HostSummary.tsx

```diff
@@ -10,6 +10,8 @@ import {
   humanHostDetailUpdated,
   wrapFleetHelper,
 } from "utilities/helpers";
+import getHostStatusTooltipText from "pages/hosts/helpers";
+import StatusCell from "components/TableContainer/DataTable/StatusCell";
```

Comment:
Add a note on line 14
// TODO: Create StatusIndicator.tsx component to be used instead